test/passing/tests/let_binding-in_indent.ml.ref

```diff
@@ -13,7 +13,7 @@ let _ =
   let (x : int) = x in
   let x : int = x in
   let (_ : int) = x in
-  let (_ : int) = x in
+  let _ : int = x in
```

Comment:
Parentheses around let-binding patterns follow the original source. These are two different AST now.
Shouldn't change existing code.